### ISSUE TITLE
signup and feedback system

### ISSUE THREAD

**Issue by suman-shah** [suman-shah/electrical-club-website#4]
create something like this one 

![image](https://user-images.githubusercontent.com/61034696/192739882-2e3d3ff1-3ac8-4c0f-a4a3-ec92963ae9fa.png)

just make these features and include it in index.html page

**Comment by codewithnadeem14502** [comment]
hello sir/mam, i want to contribute to it, can you assign me this issue and i am completely new to open source

**Comment by codewithnadeem14502** [comment]
sir , i am using html and css to build this , hope this will be fine

**Comment by codewithnadeem14502** [comment]
can you help me , to figure , what exactly you want to do in this task sir .

**Comment by suman-shah** [comment]
yes, you can use whatever you know HTML CSS bootstrap any things as your wish.  i want something like writing comments to any post or any suggestion to website owners just a little bit different than the contact option

**Comment by codewithnadeem14502** [comment]
sir , i am little bit confuse , can you share your LinkedIn profile or discord server , where we can connect .

**Comment by codewithnadeem14502** [comment]
you want data should be store permanently which the user has entered ? , this completely new to me sir , hope you will some share resource to me , thank you

**Comment by suman-shah** [comment]
i mean something like 
![image](https://user-images.githubusercontent.com/61034696/193416656-bdae96f2-35a2-4964-b3d5-058bab100810.png)

**Comment by codewithnadeem14502** [comment]
sure , sir i am working on it

**Comment by danishzayan** [comment]
## I want help..
@suman-shah Sir this is my project **YourQuotes-App** and also i am maintainer someone push code in master from master branch and there is conflicts in code but how to solve conflicts in github web and i have no idea me only resolve in vs code so how to resolve conflicts... 

Repo:  (https://github.com/danishzayan/YourQuotes-App)

![conflicts](https://user-images.githubusercontent.com/46997504/193506099-50783a4a-9bcc-4291-bcb5-3d7fad23e925.PNG)
Repo:  (https://github.com/danishzayan/YourQuotes-App)
